xio-core/src/main/java/com/xjeffrose/xio/tracing/XioTracing.java

```diff
@@ -23,11 +24,20 @@
 
   public XioTracing(TracingConfig config) {
     name = config.getApplicationName();
-    String zipkinUrl = config.getZipkinUrl();
-    float samplingRate = config.getZipkinSamplingRate();
-    Tracing tracing = buildZipkinTracing(this.name, zipkinUrl, samplingRate);
-    if (tracing != null) {
-      tracer = BraveTracer.create(tracing);
+    TracingConfig.TracingType type = config.getType();
+
+    switch (type) {
+      case ZIPKIN:
```

Comment:
maybe log the configured tracer type here

Reply:
now logging the tracer type as log.info